### DISCUSSION TITLE
Stuck on splash screen

### DISCUSSION THREAD

**Discussion by 4Sehat5Sekarat** [Acode-Foundation/Acode#922] [Q&A]
After update to latest version on FDroid version the app stuck on splash or start screen when i open this app for the first time. Idk this are stuck or just need to waiting more because i waiting for around 3 minutes.

Sory if my english bad i'm not using translator apps.

**Comment by UnderXirox** [comment]
Hello same following the last update carried out by f-droide for me the application 
does not start remains stuck on the start screen already uninstalling and reinstalling several times as well as resetting the cash on the phone nothing helps 
excuse me for my broken english Sincerely

**Reply by 4Sehat5Sekarat** [reply]
I trying using one version above and that work well. Why you dont try that?, but hopely the problem on latest version are fixed.
